<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICalendarSettingsDecl</class>
 <widget class="QWidget" name="ICalendarSettingsDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Icalendar export settings</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QCheckBox" name="kcfg_writeEventsVsTodos">
       <property name="toolTip">
        <string>Write the iCalendar file so that the entries are recognized by Google Calendar.</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Write the iCalendar file so that the entries are recognized by Google Calendar. The default is to write &lt;a href=&quot;https://icalendar.org/iCalendar-RFC-5545/3-6-2-to-do-component.html&quot;&gt;VTODO&lt;/a&gt; entries into the iCalendar file but for Google Calendar entries of type &lt;a href=&quot;https://icalendar.org/iCalendar-RFC-5545/3-6-1-event-component.html&quot;&gt;VEVENT&lt;/a&gt; are needed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Write Google calendar compatible entries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="kcfg_createAlarm">
       <property name="text">
        <string>Create reminders</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QSpinBox" name="kcfg_timeUnits">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="kcfg_timeUnitInSeconds">
         <property name="maxCount">
          <number>3</number>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="kcfg_beforeAfter"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QCheckBox" name="kcfg_repeatingReminders">
         <property name="text">
          <string comment="@action whether the schedule will have more occurrences over time">Repeating:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>How often:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>Interval:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QSpinBox" name="kcfg_numberOfReminders">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="kcfg_intervalBetweenRemindersTimeUnits">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>Time(s)</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="kcfg_intervalBetweenRemindersTimeUnitInSeconds"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="iCalendar_file">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>iCalendar file</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="kcfg_icalendarFile">
         <property name="kcfg_property" stdset="0">
          <cstring>text</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>183</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kcombobox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
